src/models/ITestType.ts

```diff
@@ -7,6 +7,10 @@ export interface ITestType {
     forVehicleSize: ForVehicleSize[] | null;
     forVehicleConfiguration: ForVehicleConfiguration[] | null;
     forVehicleAxles: number[] | null;
+    forVehicleCategory: ForVehicleCategory[] | null;
```

Comment:
Shouldn't this field be called `forEuVehicleCategory`?

Reply:
done